cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -245,51 +696,54 @@ An example of this is `CALL db.index.explicit.searchNodes('my_index','email:me*'
-| `+{parameter}+` | Syntax | Deprecated | Replaced by <<cypher-parameters, $parameter>>
-| <<functions-properties, properties()>>  | Function  | Added  |
+| `has()`  | Function  | Removed | Replaced by <<functions-exists,exists()>>
+| `str()`  | Function  | Removed | Replaced by <<functions-tostring,toString()>>
+| `+{parameter}+` | Syntax | Deprecated | Replaced by <<cypher-parameters,$parameter>>
+| <<functions-properties,properties()>>  | Function  | Added  |
 | <<query-call,CALL [\...YIELD]>>   | Clause  | Added  |
 | <<functions-point-cartesian-2d,point() - Cartesian 2D>> | Function | Added |
 | <<functions-point-wgs84-2d,point() - WGS 84 2D>> | Function | Added |
 | <<functions-distance,distance()>> | Function | Added |
-| <<java-reference#extending-neo4j-procedures, User-defined procedures>> | Functionality | Added |
-| <<functions-tostring, toString()>>   | Function  | Extended | Now also allows Boolean values as input
+| <<java-reference#extending-neo4j-procedures,User-defined procedures>> | Functionality | Added |
+| <<functions-tostring,toString()>>   | Function  | Extended | Now also allows Boolean values as input
 |===
 
 
 [[cypher-compatibility]]
 == Compatibility
 
-NOTE: The ability of Neo4j to support multiple older versions of the Cypher language has been changing.
-In versions of Neo4j before 3.5 the backwards compatibility layer included the Cypher language parser, planner and runtime.
-All supported versions of Cypher would run on the same Neo4j kernel.
-In Neo4j 3.4, however, this was changed such that the compatibility layer no longer included the runtime.
-This meant that running, for example, a `CYPHER 3.1` query inside Neo4j 3.5 would plan the query using the 3.1 planner, and run it using the 3.5 runtime and kernel.
-In Neo4j 4.0 this was changed again, such that the compatibility layer includes only the parser.
-For example, running a `CYPHER 3.5` query inside Neo4j will parse older language features, but plan using the {neo4j-version} planner, and run using the {neo4j-version} runtime and kernel.
-The primary reason for this change has been optimizations in the Cypher runtime to allow Cypher query to perform better.
+[NOTE]
+====
+Neo4j’s ability to support multiple older versions of the Cypher language has changed over time.
+In versions prior to Neo4j 3.4, the backwards compatibility layer included the Cypher language parser, planner, and runtime.
+All supported versions of Cypher ran on the same Neo4j kernel.
+However, this changed in Neo4j 3.4 when the runtime was excluded from the compatibility layer.
+When you run, e.g. a `CYPHER 3.1` query in Neo4j 3.5, the query is planned with the 3.1 planner, but run with 3.5 runtime and kernel.
+The compatibility layer changed again in Neo4j 4.0 and it now includes only the Cypher language parser.
+When you run a `CYPHER 3.5` query, e.g., in Neo4j 4.3, Neo4j parses the older language features, but uses the 4.3 planner, runtime, and kernel to plan and run the query.
```

Comment:
should we use the `{neo4j-version}` instead of explicitly saying 4.3 here?

Reply:
I think it is fine to just show an example version, if we have a generic version it could possible be non compatible later.